packages/platform-sdk-profiles/src/repositories/notification-repository.ts

```diff
@@ -8,8 +8,10 @@ interface Notification {
 	icon: string;
 	name: string;
 	body: string;
+	type: string;
 	action: string;
 	read_at?: number;
+	meta?: any;
```

Comment:
This should be `Record<string, unknown>` or `Record<string, any>` to ensure that an object is used.